tests/install/steps/test_expansion.py

```diff
@@ -90,15 +89,6 @@ def _check_node_status():
 # }}}
 # Helpers {{{
 
-def get_node_ip(node_name, ssh_config, bootstrap_config):
```

Comment:
As discussed privately, this helper is here to get _another_ node's control plane IP, not the bootstrap one. So we can't just replace it with the `control_plane_ip` fixture, unfortunately :confused: 

Also, since we need it before installing Salt on this node, we cannot use the same approach (reading `grains`). Maybe we could still use Salt-SSH and the new `metalk8s_network.get_ip_from_cidrs` method, to avoid duplication of logic, but keeping the old one should also work, so that's your call :)

Reply:
A bit overkill to use salt-ssh to retrieve IP we will use for node (so for salt-ssh) to me.
Anyway I just consider that we have only 1 CIDR for control-plane network in our tests (which is acceptable I think, at least for the moment)